shop/src/main/java/ro/msg/learning/shop/dto/StockDto.java

```diff
@@ -0,0 +1,17 @@
+package ro.msg.learning.shop.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import ro.msg.learning.shop.domain.Location;
+import ro.msg.learning.shop.domain.Product;
+@Data
+@AllArgsConstructor
+@Builder
+@NoArgsConstructor
+public class StockDto {
+    private Product product;
```

Comment:
there should be no use of an entity in a dto

Reply:
done, removed the Dto